ui/public/i18n/language.json

```
@@ -516,7 +517,8 @@
         "sender_validation": "If enabled, the sender email address (From:) must match the login user or additional addresses.",
         "lifetime": "A value of 0 means retry only once.",
         "DeleteDestination": "Delete messages in the local server that are not in the remote server. Useful for backup or pre-sync.",
-        "imapsync":"Imapsync"
+        "imapsync":"Imapsync",
+        "Exclude_Folders": "The exclusion works with regex rules, if you exclude the 'Foo' folder, all folders matching 'Foo' in the name will be excluded of synchronization. If you want to be exclusive, start the exclusion folder name by '^' and end it by '$': ^Foo$"
```

All other suggestions don't use an impersonal form.

```suggestion
        "Exclude_Folders": "The exclusion works with regular expression rules. When excluding the 'Foo' folder, all folders matching 'Foo' in the name will be excluded from the synchronization. To force an exact match, start the exclusion folder name by '^' and end it by '$' like ^Foo$". Add one entry per line.
```